tests/model_validation/tests.py

```diff
@@ -1,26 +1,15 @@
-from django.core import management
```

Comment:
I pushed an alternative to delete this test in https://github.com/django/django/pull/7102.